libraries/Microsoft.Bot.Builder.Ai.Translation/PostProcessor/PostProcessingUtilities.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Bot.Builder.Ai.Translation.PostProcessor
+{
+    internal class PostProcessingUtilities
```

Comment:
Make this public. Then write unit tests against it. :) 

Internal classes are hard to tests, and I do notice that there are no tests at all against this code.